docs/source/user_guide/performance.rst

```diff
@@ -51,10 +51,37 @@ In-depth profiling
 Under construction.
 
 
-Use CUB/cuTENSOR backends for reduction operations
---------------------------------------------------
+Use CUB/cuTENSOR backends for reduction and other routines
+----------------------------------------------------------
 
-Under construction.
+For reduction operations (such as :func:`~cupy.sum`, :func:`~cupy.prod`, :func:`~cupy.amin`, :func:`~cupy.amax`, :func:`~cupy.argmin`, :func:`~cupy.argmax`) and many more routines built upon them, CuPy ships with our own implementations so that things just work out of the box. However, there are dedicated efforts to further accelerate these routines, such as `CUB <https://github.com/NVIDIA/cub>`_ and `cuTENSOR <https://developer.nvidia.com/cutensor>`_.
+
+In order to support more performant backends wherever applicable, starting v8 CuPy introduces an environment variable :envvar:`CUPY_ACCELERATORS` to allow users to specify the desired backends (and in what order they are tried). For example, consider summing over a 256-cubic array:
+
+.. doctest::
+
+    >>> from cupyx.profiler import benchmark
```

Comment:
```
16:22:27 Document: user_guide/performance
16:22:27 --------------------------------
16:22:27 
16:22:27 Warning, treated as error:
16:22:27 **********************************************************************
16:22:27 File "user_guide/performance.rst", line 61, in default
16:22:27 Failed example:
16:22:27     from cupyx.profiler import benchmark
16:22:27 Exception raised:
16:22:27     Traceback (most recent call last):
16:22:27       File "/opt/pyenv/versions/3.8.0/lib/python3.8/doctest.py", line 1328, in __run
16:22:27         exec(compile(example.source, filename, "single",
16:22:27       File "<doctest default[0]>", line 1, in <module>
16:22:27         from cupyx.profiler import benchmark
16:22:27     ModuleNotFoundError: No module named 'cupyx.profiler'
16:22:27 
16:22:27 make: *** [doctest] Error 2
16:22:29 Makefile:176: recipe for target 'doctest' failed
```

Reply:
Yes this module only exists in v10. For v9 it's cupyx.time.repeat.

Reply:
```suggestion
    >>> from cupyx.time import repeat
```